homeassistant/components/image_processing/opencv.py

```diff
@@ -7,22 +7,56 @@
 from datetime import timedelta
 import logging
 
+import requests
+
+import voluptuous as vol
+
 from homeassistant.core import split_entity_id
 from homeassistant.components.image_processing import (
-    ImageProcessingEntity, PLATFORM_SCHEMA)
-from homeassistant.components.opencv import (
-    ATTR_MATCHES, CLASSIFIER_GROUP_CONFIG, CONF_CLASSIFIER, CONF_ENTITY_ID,
-    CONF_NAME, process_image)
+    CONF_SOURCE, CONF_ENTITY_ID, CONF_NAME, PLATFORM_SCHEMA,
+    ImageProcessingEntity)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['opencv-python==3.2.0.6', 'numpy==1.12.0']
```

Comment:
Yeah like to try import cv2 and return False is that is not possible. If you compile opencv for python it have no pip module in env, I had check that. Your pip module is only a precompile stuff for i386/amd64. But If you have a better idea, I'm open for that.